vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -51,42 +51,41 @@ A customized image can include:
 * New APIs 
 * Legacy Entando plugins
 * New database tables  
-* Other extensions to the **entando-core** 
+* Other extensions to the entando-core
 
-It is recommended that most extensions to the Entando Platform occur in microservices. However, legacy 
+It is recommended that most extensions to the Entando Platform are made in the microservices. However, legacy 
 integrations, extensions to the CMS, and migrations from earlier Entando versions may require changes to the **entando-de-app**. 
 
 ## App Builder
-The App Builder is the user-friendly frontend UI for the **entando-de-app**. A ReactJS application, it is served via node in the default deployment. In a quickstart deployment, the App Builder container is deployed in the **entando-app** multi-container pod. It communicates with the **entando-de-app** and the Entando Component Manager (ECM) via [REST 
-APIs](../consume/entando-apis.md).  The ECM provides information about bundles deployed to the Local Hub.
+The **App Builder** is the user-friendly frontend UI for the `entando-de-app`. A ReactJS application, it is served via node in the default deployment. In a quickstart deployment, the App Builder container is deployed in the EntandoApp multi-container pod. It communicates with the `entando-de-app` and the Entando Component Manager via [REST 
```

Comment:
Multi-container is no longer correct. Probably just remove the 3rd sentence since AB gets its own container which is kind of expected and the sentence doesn't provide any additional information.